sfcasts/ep1/tdd.md

```diff
@@ -127,15 +141,17 @@ tests again.
 And... we've done it! That's TDD - write the test, see the test fail,
 write simple code to see the test pass, then refactor our code. Rinse and repeat.
```

Comment:
Maybe add:

> TDD is interesting because, by writing the test first, it forces you to think about *exactly* how a feature should work... instead of just blindly writing code and seeing what comes out. It also helps you focus on *what* you need to code... without making things too fancy. Get your tests to pass, then refactor. Nothing more is needed.